<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cam2ascii</class>
 <widget class="QMainWindow" name="Cam2ascii">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cam2ascii</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="appFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	
	background-color: rgb(47, 47, 47);
	color: white;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(255, 255, 255)
}</string>
     </property>
     <property name="text">
      <string>$&gt; &lt;strong&gt;CAM&lt;/strong&gt;2ascii</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
    <widget class="QComboBox" name="methodsComboBox">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>170</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	border-style: none;
	background-color: rgb(80, 80, 80);
	color:white;
	text-align: center;
	padding: 0 15px;
}

QComboBox::drop-down{
	width:20px;
}

QComboBox QAbstractItemView {
      border: 2px solid red;
      selection-background-color: blue;
}

QComboBox QAbstractItemView::item {
      border: 2px solid darkgray;
}
</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="methodsLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(255, 255, 255)
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;$&amp;gt; Methods_&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="charListLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(255, 255, 255)
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;$&amp;gt; Char list_&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
    </widget>
    <widget class="QComboBox" name="charListComboBox">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>280</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	border-style: none;
	background-color: rgb(80, 80, 80);
	color:white;
	text-align: center;
	padding: 0 15px;
}

QComboBox::drop-down{
	width:20px;
}

QComboBox QAbstractItemView {
      border: 2px solid red;
      selection-background-color: blue;
}

QComboBox QAbstractItemView::item {
      border: 2px solid darkgray;
}
</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string notr="true"/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCharList">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>340</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border-style: none;
	background-color: rgb(80, 80, 80);
	color:white;
	text-align: center;
	padding: 0 15px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addCharList">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>340</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: none;
	background-color: rgb(80, 80, 80);
	color:white;
	text-align: center;
	border-radius:4px
}

QPushButton::hover {
	
	background-color: rgb(121, 121, 121);
}

QPushButton::pressed {
	
	
	background-color: rgb(98, 98, 98);
}</string>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QLabel" name="optionsLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(255, 255, 255)
}</string>
     </property>
     <property name="text">
      <string>$&gt; Options_</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
    </widget>
    <widget class="QSlider" name="horRezSlider">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>450</y>
       <width>245</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal { 
	background-color:rgb(80, 80, 80);
	height: 10px; 
	border-radius: 4px;
}


QSlider::handle:horizontal { 
	background-color: rgb(135, 135, 135);
	width: 8px; 
	line-height: 20px; 
	margin-top: -5px; 
	margin-bottom: -5px; 
	border-radius: 4px; 
}

QSlider::handle:horizontal:hover { 
	background-color: white;
}</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>266</number>
     </property>
     <property name="sliderPosition">
      <number>80</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="horRezLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>450</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(255, 255, 255)
}</string>
     </property>
     <property name="text">
      <string>Resolution</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="fpsSpinBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>490</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox{
	border-style: none;
	background-color: rgb(80, 80, 80);
	color:white;
	padding:0 5px;
	
}</string>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="fpsLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>490</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(255, 255, 255)
}</string>
     </property>
     <property name="text">
      <string>Frame rate</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="mirrorCheckBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>530</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="mirrorLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>528</y>
       <width>110</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(255, 255, 255)
}</string>
     </property>
     <property name="text">
      <string>Mirror image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="screenAreaFrame">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>800</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	
	background-color: rgb(100, 100, 100);
	color:white;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="screenArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>720</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="printInfoLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>700</y>
       <width>800</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>screenAreaFrame</zorder>
   <zorder>appFrame</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
